### ISSUE TITLE
Some issues with Brave

### ISSUE THREAD

**Issue by 4aficiona2** [lyne-design-system/lyne-components#50]
## Environment

> Type of device (e.g. Laptop with Touchscreen)

> Operating system (e.g. Windows 10)

> Browser / browser version (e.g. Mozilla Firefox 74.0 (64-bit))

> Automated testing tool and rule set (e.g. axe - Web Accessibility Testing Extension)

> Assistive technology and version used to verify (e.g. NVDA version 2019.3.1)

> Additional settings (e.g. Windows High Contrast Mode)

## Which package(s) are you using?

- [x] `lyne-components`
- [x] `lyne-design-tokens`
- [ ] `non package issue`

## Detailed description

> Which [version](https://lyne-components-deployments.netlify.app/) of *Lyne Components* are you using?

> Is the issue related to a specific component? If so please provide the components Storybook link of the corresponding version.

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

> Is the issue a general finding or specifically related to the usage of *Lyne Components* 
> within your application?

> If the latter, can you provide us with a link to your application where the behaviour can 
> be seen or replicated?

> Since when has the issue been noticed? Can it be related to a specific 
> *Lyne Components* version change?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - https://codesandbox.io/

## Additional information

- Screenshots or code
- Notes
